l1-contracts/contracts/state-transition/StateTransitionManager.sol

```diff
@@ -231,6 +231,9 @@ contract StateTransitionManager is IStateTransitionManager, ReentrancyGuard, Own
         uint256 _chainId,
         address _stateTransitionContract
     ) external onlyOwner {
+        require(_chainId != 0, "STM: chainId zero");
```

Comment:
I think that there were no such recommendation

Reply:
Yes, this is true. I misunderstood initially.